### ISSUE TITLE
keyExists does not check partial fallback for regional locals while $t does

### ISSUE THREAD

**Issue by BlakeHancock** [dkfbasel/vuex-i18n#57]
If I have the following locales added and my current locale is en-US.
```
{
    "en": {login: "Login"},
    "en-US": {},
    "en-CA": {}
}
```

`$i18n.keyExists('login')` will return false but `$t('login')` will still render as expected.

I was looking to do something like this

```
<p v-if="$i18n.keyExists(key)">
    {{ $t(key) }}
</p>
```

**Comment by tikiatua** [comment]
Hi @BlakeHancock,

Thank you for reporting this. The problem would be on what would be acceptable to define as existing key. Should we also include the fallback locale or only the current language localizations? 

Maybe it would be best to keep the method as it is and provide a second argument to define what scope to search the key in. 

```javascript
// strict would be the default and only search in the current locale
$i18n.keyExists(key, 'strict')

// locale would also search all same language localizations
$i18n.keyExists(key, 'locale')

// locale would search the same language localization and the fallback
$i18n.keyExists(key, 'fallback')
```

**Comment by BlakeHancock** [comment]
The second argument idea works. For the default though, I would think it would make most sense to follow the same behavior as $t, but that could potentially be backwards incompatible with some peoples projects.

**Comment by tikiatua** [comment]
The compatibility issue was the reason I proposed to use strict as default. In hindsight I think it is better to use the same logic as for $t and provide the second argument to limit the scope. 

We will just publish this as 1.9 and write a respective comment. The fix will be available some time this afternoon.

**Comment by tikiatua** [comment]
Hi @BlakeHancock 

The issue should be fixed now in v1.9.0. It would be great, if you could test it and give us some feedback.

**Comment by BlakeHancock** [comment]
I have updated to v1.9.0 in my project and everything appears to be good and working.

Thanks for the quick turn around.
